### PR TITLE
aws-cloudformation: add page

### PR DESCRIPTION
**Pull request by kxrtiswithak**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/aws-cloudformation.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Check the status of a stack:
+
+`aws cloudformation describe-stacks --stack-name {{stack-id}} --profile {{profile}}`
+
+- Initiate drift detection for a stack:
+
+`aws cloudformation detect-stack-drift --stack-name {{stack-id}} --profile {{profile}}`
+
+- Check drift status output of a stack using 'StackDriftDetectionId' from previous command output:
```
Comment (kbdharun) [inline pages/common/aws-cloudformation.md:30]:
```suggestion
- Check the drift status output of a stack using 'StackDriftDetectionId' from the previous command output:
```

File: pages/common/aws-cloudformation.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# aws cloudformation
+
+> AWS CloudFormation lets you model, provision, and manage AWS and third-party resources by treating infrastructure as code.
```
Comment (kbdharun) [inline pages/common/aws-cloudformation.md:3]:
```suggestion
> Model, provision, and manage AWS and third-party resources by treating infrastructure as code.
```
How about this? (the title already suggests the same.)

**Review by pixelcmtd — APPROVED** [review]
Once the comment above has been addressed

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**